clang/lib/Sema/HLSLExternalSemaSource.cpp

```diff
@@ -483,10 +573,103 @@ static BuiltinTypeDeclBuilder setupBufferType(CXXRecordDecl *Decl, Sema &S,
       .addDefaultHandleConstructor(S, RC);
 }
 
+BinaryOperator *getSizeOfLEQ16Expr(clang::ASTContext &context,
+                                   SourceLocation NameLoc,
+                                   TemplateTypeParmDecl *T) {
+  // Obtain the QualType for 'unsigned long'
+  clang::QualType unsignedLongType = context.UnsignedLongTy;
```

Comment:
There's a `using namespace clang` at the beginning of this cpp file, so I don't think any of these `clang::` qualifications are necessary. A quick survey of the rest of the existing code suggests that they are usually omitted.

Reply:
There's some remaining in getSizeOfLEQ16Expr if you wanted to also update that.